Makefile

```diff
@@ -264,6 +262,7 @@ test-unit-js: test-unit-graphql-endpoint
 test-unit-js: ## Build and run unit tests - JavaScript only
 
 .PHONY: test-unit-graphql-endpoint
+test-unit-graphql-endpoint: | dist
```

Comment:
What does the pipe in front of the dist target do?

Reply:
It marks `dist` as an [order-only prerequisite](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html) for this target. This is basically the prototypical usecase for such prerequisites (detailed toward the bottom of the linked page).